addons/controllers/tanzuclusterbootstrap_controller.go

```diff
@@ -498,22 +499,28 @@ func (r *TanzuClusterBootstrapReconciler) updateValuesFromSecret(cluster *cluste
 				UID:        cluster.UID,
 			},
 		}
-		newSecret.Name = fmt.Sprintf("%s-%s-package", cluster.Name, packageShortName)
+		// Add cluster and package labels to cloned secrets
+		newSecret.Labels = map[string]string{
```

Comment:
Nit: Do you think we need to do something like following, so that we don't overwrite the labels added by others.
```
if newSecret.Labels == nil {
    newSecret.Labels = map[string]string{}
}
newSecret.Labels[PackageNameLabel] = pkg.RefName
newSecret.Labels[ClusterNameLabel] = cluster.Name
```

Reply:
sounds a safe thing to do.